app/views/home/apilist.html.haml

```diff
@@ -0,0 +1,56 @@
+.api-section-top-margin.grey-section.diagonal
+  .home-content
+    %h1 Brickhack APIs
+    .split-three
+      .column
+        = image_tag 'sponsors/microsoft.png', class: 'img-spaced'
+        %h2 Microsoft Cognitive APIs
+        %p Tinker with the some of the Cognitive APIs of one of the largest tech giants in the world.
+        %p It's building time. Integrate these vision, speech, language, knowledge, and search APIs into your app, on the platform of your choice.
+        = btn_link_to 'Check it out', 'https://www.microsoft.com/cognitive-services/en-us/apis'
+      .column
+        = image_tag 'sponsors/ibm.png', class: 'img-spaced'
+        %h2 IBM Watson
+        %p Go beyond artificial intelligence with Watson.
+        %p With Watson APIs and solutions, businesses are already achieving outcomes – from improving customer engagement, to scaling expertise, to driving innovation and growth.
+        = btn_link_to 'Check it out', 'https://www.ibm.com/watson/'
+      .column
+        = image_tag 'sponsors/ups.png', class: 'img-spaced'
+        %h2 UPS Developer Kit
+        %p Whether it's pre-shipment or tracking, UPS Developer Kit APIs can help provide a seamless, positive customer experience.
+        %p The UPS Developer Kit APIs offer various services to enhance shipping and tracking systems. For all rate and service changes or enhancements, refer to the UPS Rate and Service Guide. For all technical changes and enhancements, refer to the Developer's Guide specific to each API.
+        = btn_link_to 'Check it out', 'http://ups.com/upsdeveloperkit'
+
```

Comment:
Can we have the logos & headings line up horizontally?

![screenshot 2017-04-23 22 48 47](https://cloud.githubusercontent.com/assets/1441807/25320708/4866e786-2877-11e7-8e84-ee9829a0e376.png)